src/Types/ConcordiumBftDetails.cs

```diff
@@ -0,0 +1,33 @@
+using Concordium.Sdk.Helpers;
+
+namespace Concordium.Sdk.Types;
+
+/// <summary>
+/// Parameters pertaining to the Concordium BFT consensus.
+/// </summary>
+/// <param name="CurrentTimeoutDuration">The current duration to wait before a round times out.</param>
+/// <param name="Round">The current round.</param>
+/// <param name="Epoch">The current epoch.</param>
+/// <param name="TriggerBlockTime">
+/// The first block in the epoch with timestamp at least this is considered
```

Comment:
Consider rephrasing to:

_The first block in the epoc with **a** timestamp **equal to or later than this timestamp,** is considered ...._